integration/postinit.sh

```diff
@@ -23,5 +23,6 @@ mv $GENESIS.tmp $GENESIS
 
 
 echo "Starting allorad daemon and sleep for 3 seconds to let it start"
-$ALLORAD_BIN start & disown;
-sleep 3
+# $ALLORAD_BIN start & disown;
+# sleep 3
+$ALLORAD_BIN start
```

Comment:
why are you starting this without disowning, also the sleeping affects stability as well

Reply:
sorry - that slipped in. I used that for my local testing without it. I'll change it. good catch.

Reply:
done